test/functional/import-abort-rescan.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test wallet import RPCs.
+
+Test rescan behavior of importprivkey when aborted. The test ensures that:
+1. The abortrescan command indeed stops the rescan process.
+2. Subsequent rescan catches the aborted address UTXO
+"""
+
+from decimal import Decimal
+import threading # for bg importprivkey
+import time      # for sleep
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (assert_equal, get_rpc_proxy)
+
+class ImportAbortRescanTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 2
+
+    def run_test(self):
+        # Generate for BTC
+        self.nodes[0].generate(300)
+        self.log.info("Creating blocks with transactions ...")
```

Comment:
@jnewbery wdyt about moving this to `util.py`, like `def generate_load(node, blocks, transactions):` or whatever?

Reply:
I don't think it makes sense to move code up into the `test_framework` directory unless it's being used by more than one test script. I don't see the need here (unless you think you can refactor other test scripts to use this and reduce LOC)

Reply:
Would it make sense to take advantage of `mine_large_block` or is that overkill in this context? Might make this test more concise.